dev/com.ibm.ws.jsfContainer_fat_2.2/bnd.bnd

```diff
@@ -33,5 +33,10 @@ fat.project: true
 	com.ibm.websphere.javaee.jsf.2.2;version=latest,\
```

Comment:
Should we update the copyright date range?

Reply:
updated years